HACKING.txt

```diff
@@ -161,15 +145,15 @@ Coding Style
 Running Tests
 --------------
 
-- To run all tests for Pyramid on a single Python version, run ``nosetests``
-  from your development virtualenv (See *Using a Development Checkout* above).
+- To run all tests for Pyramid on a single Python version from your development
+  virtual environment (See *Using a Development Checkout* above), and assuming
+  you have installed the test requirements::
+
+   $ $VENV/bin/py.test --strict pyramid/
```

Comment:
No, we are not changing the recommendation in pyramid away from nose. Based on the discussion in https://github.com/Pylons/pyramid/issues/2474, statements like these need to be fixed.